types/stripe/index.d.ts

```diff
@@ -5,11 +5,12 @@
 //                 Amrit Kahlon <https://github.com/amritk/>
 //                 Adam Cmiel <https://github.com/adamcmiel>
 //                 Justin Leider <https://github.com/jleider>
+//                 Kamil Gałuszka <https://github.com/galuszkak>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
```

Comment:
Its pretty hacky, I have one version installed through npm @types and one from typings